### PR TITLE
Update Banners / Versions to Beta

### PR DESCRIPTION
**Pull request by dternyak**

### CODE REVIEW

**Comment by dternyak** [comment]
I left the header $brand-warning - the blue doesn't work well.
screen shot 2018-02-09 at 2 01 11 pm

File: common/config/data.tsx [hunk 1]
```diff
@@ -13,11 +13,11 @@ export const N_FACTOR = 8192;
 export const ANNOUNCEMENT_TYPE = 'warning';
 export const ANNOUNCEMENT_MESSAGE = (
   <div>
-    This is an Beta build of MyCrypto v4. Please only use for testing, or use production at{' '}
+    This is an Beta build of MyCrypto Beta. Please only use for testing, or use production at{' '}
```
Comment (wbobeirne) [inline common/config/data.tsx]: *a Beta build. And maybe this is just me, but a beta build of the beta reads a little redundantly to me.

Comment (wbobeirne) [reply common/config/data.tsx]: Maybe "pre-release" build or "early" build or "preview" build.

Comment (dternyak) [reply common/config/data.tsx]: definitely, good catch

**Pull request merged**